python/lib/dcoscli/dcoscli/metrics.py

```diff
@@ -95,13 +95,30 @@ def _node_summary_json(datapoints):
     :return: JSON data
     :rtype: str
     """
-    summary_datapoints = [
-        _get_datapoint(datapoints, 'cpu.total'),
-        _get_datapoint(datapoints, 'memory.total'),
-        _get_datapoint(datapoints, 'memory.free'),
-        _get_datapoint(datapoints, 'filesystem.capacity.total', {'path': '/'}),
-        _get_datapoint(datapoints, 'filesystem.capacity.used', {'path': '/'})
+
+    metrics = [
+        ['cpu.total'],
+        ['memory.total'],
+        ['memory.free'],
+        ['filesystem.capacity.total', {'path': '/'}],
+        ['filesystem.capacity.used', {'path': '/'}],
     ]
+
+    summary_datapoints = []
+    for m in metrics:
+        datapoint = _get_datapoint(datapoints, *m)
+        if datapoint is None:
+            datapoint = {
+              "name": m[0],
+              "value": "N/A",
+              "unit": "N/A",
+              "timestamp": "N/A"
+            }
```

Comment:
When the datapoint is missing, is there any reason for adding an entry with `N/A` fields instead of omitting it from the list? This might break some scripts looping over the json and trying to parse units/value/timestamp as `N/A` is not a standard / well-defined way to indicate missing data.